be/src/runtime/lib_cache.h

```diff
@@ -161,6 +161,36 @@ class LibCache {
         const std::string& hdfs_lib_file, LibType type,
         boost::unique_lock<boost::mutex>* entry_lock, LibCacheEntry** entry);
 
+    // map "palo" to "doris" in symbol, only for grayscale upgrading
+    std::string get_real_symbol(const std::string& symbol) {
+        const std::string& str1 = replace_all(symbol, "8palo_udf", "9doris_udf");
+        const std::string& str2 = replace_all(str1, "4palo", "5doris"); 
```

Comment:
you can't use reference here, just use std::string, complier will do RVO

Reply:
oh, variables in the stack will be released , when function finished. I forget , i will correct it.